.github/workflows/release.yaml

```diff
@@ -41,27 +41,31 @@ jobs:
         with:
           name: dist
 
-      - name: Download Gon
+      - name: Import Code-Signing Certificates
+        uses: Apple-Actions/import-codesign-certs@v1
+        with:
+          # The certificates in a PKCS12 file encoded as a base64 string
+          p12-file-base64: ${{ secrets.INSTALLER_CERT_MAC_P12 }}
+          # The password used to import the PKCS12 file.
+          p12-password: ${{ secrets.INSTALLER_CERT_MAC_PASSWORD }}
+
+      - name: Install gon via HomeBrew for code signing and app notarization
         run: |
-          wget -q https://github.com/mitchellh/gon/releases/download/v0.2.2/gon_0.2.2_macos.zip
-          unzip gon_0.2.2_macos.zip -d /usr/local/bin
-          rm -f gon_0.2.2_macos.zip
+          brew tap mitchellh/gon
+          brew install mitchellh/gon/gon
```

Comment:
@umbynos nice addition! Is it possible to pin the `gon` version?

Reply:
With homebrew taps apparently is not possible, I reverted to the previous installation method